test/libsolidity/syntaxTests/literalSuffixes/parameters/signed_exponent.sol

```diff
@@ -0,0 +1,18 @@
+function s8(uint mantissa, int8 exponent) pure returns (uint) { }
+function s64(uint mantissa, int64 exponent) pure returns (uint) { }
+function s160(uint mantissa, int160 exponent) pure returns (uint) { }
+function s256(uint mantissa, int256 exponent) pure returns (uint) { }
```

Comment:
Can you add some variants with different mantissa types, especially signed ones?

Reply:
Done.